shared/desktop/app/app-state.js

```diff
@@ -45,36 +46,49 @@ export default class AppState {
   config: Config
   managed: Managed
 
-  constructor(opts: Options) {
+  constructor() {
     this.state = {
-      x: null,
-      y: null,
-      width: opts.defaultWidth,
-      height: opts.defaultHeight,
-      windowHidden: false,
-      isMaximized: null,
-      isFullScreen: null,
       displayBounds: null,
-      tab: null,
       dockHidden: false,
-      openAtLoginSet: false,
+      height: windowStyle.height,
+      isFullScreen: null,
+      isMaximized: null,
+      openAtLogin: true,
+      tab: null,
+      width: windowStyle.width,
+      windowHidden: false,
+      x: null,
+      y: null,
     }
 
     this.config = {
+      eventHandlingDelay: 1000,
```

Comment:
What does this `100` -> `1000` do?

Reply:
i upped it since we don't need to be updating this state or making these calls that quickly. 100 ms is pretty short for us writing to this file all the time (as your dragging windows around etc)

Reply:
Ah cool :+1: